### ISSUE TITLE
OACMPDM2 env seems

### ISSUE THREAD

**Issue by Bishop-333** [NeonKnightOA/oacmpvol1#40]
I don't really know if it's considered as an issue but there seems to the sky and I think it's the env of akom called nebulae2 specific to the mappack :
![shot0031](https://user-images.githubusercontent.com/99912467/162948353-6f446d2e-da71-426e-b007-a1cfed4fa182.jpg)

**Comment by EddieBrrrock** [comment]
I think I can help with that

**Comment by EddieBrrrock** [comment]
z_oacmp1v4_beta-220411

**Comment by NeonKnightOA** [comment]
That looks like the texture needs to be worked on.

**Comment by NeonKnightOA** [comment]
That looks like a texture that doesn't tile. Guess I need to check.

**Comment by NeonKnightOA** [comment]
https://mega.nz/file/bMZkRJBa#J_rUm2nz2SpiMFl6kBY0eMceeCLOJ1PLfnz_-CQRzCw
New test version. I didn't notice those seams, may be a problem with the shaders.

**Comment by NeonKnightOA** [comment]
Since this isn't going anywhere, closing.

**Comment by The-Gig** [comment]
Sorry for the late reply.

Screenshots from version 230430:
![shot0004](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/c3f16ebf-e4f2-4356-9749-241756739b4c)
Issue not very noticeable, but indeed there. Clearer if zooming.
![shot0005](https://github.com/NeonKnightOA/oacmpvol1/assets/23258514/6e03be12-75aa-4c97-87e9-9eef45adf642)
Not sure whether it's a serious "problem", I had to search for it to see it. But I guess it may get more visible the highest brightness you have.
And I'm not sure about how to fix. I thought maybe nebuae2\* images weren't "tileable" well, but I tried pasting 4 copies of textures\akom_env\nebuale2_bk.jpg next each other in an image editor and the seams didn't look so noticeable to me... I don't know.
